README.md

```diff
@@ -1,40 +1,133 @@
-# Canvas Course Inventory
+# course-inventory
+
+
+## Overview
+
+The course-inventory application is designed to gather current-term Canvas LMS data about courses, enrollments, and users in order to inform leadership at the University of Michigan about the status and utilization of Canvas courses. Currently, the application collects data from the Canvas API and the Unizin Data Warehouse and then stores the data in an external MySQL database. Tableau dashboards and other processes then consume that data to generate reports and visualizations.
+
+
+## Development
+
+
+### Pre-requisities
+
+The sections below provide instructions for configuring, installing, using, and changing the application. Depending on the environment you plan to run the application in, you may also need to install some or all of the following:
+   * [Python 3.7](https://docs.python.org/3/)
+   * [MySQL](https://dev.mysql.com/doc/)
+   * [Docker Desktop](https://www.docker.com/products/docker-desktop)
+   * [OpenShift CLI](https://docs.openshift.com/enterprise/3.1/cli_reference/get_started_cli.html)
+
+While performing any of the actions described below, use a terminal, text editor, or file utility as necessary. Some sample command-line instructions are provided for some steps.
+
+
+### Configuration
+
+To configure the application before installation and usage (see the next section), you must first perform a few steps, including the creation of a configuration file called `env.json`. Complete the following items in order.
+
+1. Clone and navigate into the repository.
+    ```
+    https://github.com/tl-its-umich-edu/course-inventory.git  # HTTPS
```

Comment:
specify out the "git clone" command, since other code snippets are complete code lines.

Reply:
Wups

Reply:
Fixed this.